config/objects/dwellings.json

```diff
@@ -2,6 +2,7 @@
 	"creatureGeneratorCommon" : {
 		"index" :17,
 		"handler": "dwelling",
+		"ambientSound" : "LOOPSWOR",
```

Comment:
As you adding sounds on sybtype level, do not add it on top level. Add default sound in base struct below.